pkgs/top-level/python-packages.nix

```diff
@@ -7915,6 +7915,8 @@ self: super: with self; {
 
   mkdocs-git-authors-plugin = callPackage ../development/python-modules/mkdocs-git-authors-plugin { };
 
+  mkdocs-git-committers-plugin-2 = callPackage ../development/python-modules/mkdocs-git-committers-plugin-2 { };
```

Comment:
I'm a bit rusty on Python packaging, since those are not libraries could they not go under the by-name overlay?

Reply:
A little unsure myself, but since they don't expose any binaries and their counterparts are also listed here, I figured this would be the correct location? Not too sure, though.